phpunit/block-supports/border-test.php

```diff
@@ -117,7 +117,7 @@ function test_border_color_slug_with_numbers_is_kebab_cased_properly() {
 		$actual   = gutenberg_apply_border_support( $block_type, $block_atts );
 		$expected = array(
 			'class' => 'has-border-color has-red-border-color',
-			'style' => 'border-radius: 10px; border-style: dashed; border-width: 1px;',
```

Comment:
Personally, I'd prefer to leave the spaces out as you have them here.

But they've been around forever, so I'd tend towards leaving them in for now and making the cosmetic change down the line. Or maybe we could add a `prettify` option to an inline style generator as well.

Reply:
Makes sense I suppose. Done 👍